src/hats/catalog/dataset/table_properties.py

```diff
@@ -130,12 +130,15 @@ def space_delimited_list(cls, str_value: str) -> List[str]:
         if isinstance(str_value, str):
             # Split on a few kinds of delimiters (just to be safe), and remove duplicates
             return list(filter(None, re.split(";| |,|\n", str_value)))
-        return str_value
+        if str_value:
+            return str_value
+        ## Convert empty strings and empty lists to None
+        return None
```

Comment:
I like the ternary, but it's just my preference.
```suggestion
        ## Convert empty strings and empty lists to None
        return str_value if str_value else None
```

Reply:
Dig it.